pkg/tfgen/test_data/regress-611-schema.json

```diff
@@ -213,7 +213,7 @@
         "values"
       ]
     },
-    "aws:x/getPolicyDocumentStatement:getPolicyDocumentStatement": {
+    "aws:x/iam/getPolicyDocumentStatement:getPolicyDocumentStatement": {
```

Comment:
@viveklak PTAL - I've rebased this on https://github.com/pulumi/pulumi-terraform-bridge/pull/624 so now we can see how the changes affect the schema generation. The only diff is visible here is the expected diff - "x" becomes "x/iam". The other types and functions in the schema add some confidence that the rest of the schema generation does not regress.